src/lib.rs

```diff
@@ -1,5 +1,7 @@
 mod de;
 mod error;
+pub mod test_struct;
```

Comment:
please remove this module and move content inside tests, even if it means code duplication